src/index.js

```diff
@@ -17,7 +17,10 @@ export type MongoOptionsType = {
     updatedAt: string,
   },
   convertUserIdToMongoObjectId: boolean,
-  caseSensitiveUserName: boolean
+  convertSessionIdToMongoObjectId: boolean,
+  caseSensitiveUserName: boolean,
+  idProvider: ?(() => string | Object),
```

Comment:
This function provides id for session only.
We should either use the id provider in the users collection (probably preferable) or rather change the name to `sessionIdProvider` and then add also `userIdProvider`